osu.Game/Screens/Play/DimmableStoryboard.cs

```diff
@@ -13,6 +14,8 @@ namespace osu.Game.Screens.Play
     /// </summary>
     public class DimmableStoryboard : UserDimContainer
     {
+        public Container OverlayLayerContainer;
```

Comment:
nit: wouldn't it be better to expose a `CreateOverlayLayerProxy()` method instead? (with a xmldoc explaining that it has to be proxied at a `Player`-level above the gameplay and below the HUD)

Reply:
I'm not too convinced. Nothing else in `Player` does anything like this.